### PR TITLE
Remove the extra "/model_zoo" when copy the model zoo files. 

### LINKED ISSUE #1740: Copy the user defined model zoo files into root  in the docker image.
Sometimes, we need to define multiple files for a model. For example:
```
model_zoo
    |----mnist_model
        |----__init__.py
        |----mnist_model_functional.py
        |----custom_layer.py
```
We can import the module from `custom_layer.py` in `mnist_model_functional.py` using
`from model_zoo.custom_layer import CustomLayer`.  

However, we copy the `model_zoo` directory into `model_zoo/model_zoo` when we build the image. And the directory structure is
```
model_zoo
    |----model_zoo
        |----mnist_model
            |----__init__.py
            |----mnist_model_functional.py
            |----custom_layer.py
```
So, the user must use `from model_zoo.model_zoo.custom_layer import CustomLayer` to import.

### PR DESCRIPTION
**Pull request by workingloong**
Fix #1740

### CODE REVIEW

**Review by terrytangyuan — APPROVED** [review]
LGTM as long as all existing examples pass

File: elasticdl/python/elasticdl/image_builder.py [hunk 1]
```diff
@@ -154,14 +154,14 @@ def _create_dockerfile(
 RUN pip install -r /elasticdl/requirements.txt \
   --extra-index-url="${EXTRA_PYPI_INDEX}"
 RUN make -f /elasticdl/Makefile
-COPY {MODEL_ZOO} /model_zoo/{MODEL_ZOO}
+COPY {MODEL_ZOO} /{MODEL_ZOO}
```
Comment (skydoorkai) [inline elasticdl/python/elasticdl/image_builder.py:157]: I think the correct way is that in image_builder, we copy `model_def` into `/model_zoo`. 
For example, on the client, `cifar10_functional_api` is in `some_dir`. The user set `model_zoo=some_dir`.
After image_builder, `cifar10_functional_api` is under `/model_zoo` in the docker image.

Then in master pod argument, set `model_zoo=/model_zoo`.

Comment (workingloong) [reply elasticdl/python/elasticdl/image_builder.py:157]: If `cifar10_functional_api` is in `some_dir`, the user will write 
`from some_dir.cifar10_function_api.some_module import some_class` to import `some_class` from `some_module.py`. 
If `cifar10_functional_api` is under `/model_zoo` in the docker image, the import error will happen and the corret import code in the image is
`from model_zoo.cifar10_function_api.some_module import some_class`

File: scripts/client_test.sh [hunk 2]
```diff
@@ -65,7 +65,7 @@ elif [[ "$JOB_TYPE" == "predict" ]]; then
       --image_base=elasticdl:ci \
       --model_zoo=model_zoo \
       --model_def=mnist_functional_api.mnist_functional_api.custom_model \
-      --checkpoint_dir_for_init=model_zoo/${MNIST_CKPT_DIR}/version-100 \
+      --checkpoint_dir_for_init=/${MNIST_CKPT_DIR}/version-100 \
```
Comment (brightcoder01) [inline scripts/client_test.sh:68]: We can also update the example command in elasticdl_local.md and elasticdl_on_prem_cluster.md.
The model zoo parameter is set as follows, we need update them too
```
 --model_zoo=/model_zoo/model_zoo \
```

File: elasticdl/python/elasticdl/image_builder.py (hunk 1 above)
Comment (skydoorkai) [reply elasticdl/python/elasticdl/image_builder.py:157]: ok, then I assume the user will use the root directory of `some_dir` as python directory.

File: scripts/client_test.sh (hunk 2 above)
Comment (workingloong) [reply scripts/client_test.sh:68]: Yes!

**Pull request merged**